### ISSUE TITLE
Crash when opening full screen comment editor from Reader

### ISSUE THREAD

**Issue by khaykov** [wordpress-mobile/WordPress-Android#11135]
@frosty reported the issue that affects our `develop` branch, and not yet present in the Store version.

Looks like it might have been introduced with [this](https://github.com/wordpress-mobile/WordPress-Android/commit/66b96db8b4ebce5d59a9f19012718ef4c0beb0aa#diff-e17b9315039f65e393cdc6ee860af3b5) commit. cc @hafizrahman 

It does not happen with all posts.

Navigate to reader or post, tap on comment icon and expand comment editor.

[![Image from Gyazo](https://i.gyazo.com/307869d721d975dcc4cc5cb6b3cf4767.gif)](https://gyazo.com/307869d721d975dcc4cc5cb6b3cf4767)

```
2020-01-22 14:39:56.298 18762-18762/org.wordpress.android.beta E/AndroidRuntime: FATAL EXCEPTION: main
    Process: org.wordpress.android.beta, PID: 18762
    java.lang.IllegalStateException: siteStore.getSiteBySiteI…t.getLong(EXTRA_SITE_ID)) must not be null
        at org.wordpress.android.ui.CommentFullScreenDialogFragment.onCreateView(CommentFullScreenDialogFragment.kt:66)
        at androidx.fragment.app.Fragment.performCreateView(Fragment.java:2439)
        at androidx.fragment.app.FragmentManagerImpl.moveToState(FragmentManager.java:1460)
        at androidx.fragment.app.FragmentManagerImpl.moveFragmentToExpectedState(FragmentManager.java:1784)
        at androidx.fragment.app.FragmentManagerImpl.moveToState(FragmentManager.java:1852)
        at androidx.fragment.app.FragmentManagerImpl.dispatchStateChange(FragmentManager.java:3269)
        at androidx.fragment.app.FragmentManagerImpl.dispatchActivityCreated(FragmentManager.java:3229)
        at androidx.fragment.app.Fragment.performActivityCreated(Fragment.java:2466)
        at androidx.fragment.app.FragmentManagerImpl.moveToState(FragmentManager.java:1483)
        at androidx.fragment.app.FragmentManagerImpl.moveFragmentToExpectedState(FragmentManager.java:1784)
        at androidx.fragment.app.FragmentManagerImpl.moveToState(FragmentManager.java:1852)
        at androidx.fragment.app.BackStackRecord.executeOps(BackStackRecord.java:802)
        at androidx.fragment.app.FragmentManagerImpl.executeOps(FragmentManager.java:2625)
        at androidx.fragment.app.FragmentManagerImpl.executeOpsTogether(FragmentManager.java:2411)
        at androidx.fragment.app.FragmentManagerImpl.removeRedundantOperationsAndExecute(FragmentManager.java:2366)
        at androidx.fragment.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:2273)
        at androidx.fragment.app.FragmentManagerImpl$1.run(FragmentManager.java:733)
        at android.os.Handler.handleCallback(Handler.java:883)
        at android.os.Handler.dispatchMessage(Handler.java:100)
        at android.os.Looper.loop(Looper.java:214)
        at android.app.ActivityThread.main(ActivityThread.java:7356)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)
```

**Comment by hafizrahman** [comment]
Looking!

**Comment by shiki** [comment]
Thanks for reporting this, @khaykov and @frosty. And thanks to @hafizrahman for the quick turnaround. The fix will be merged with `release/14.0` in #11149.

**Comment by designsimply** [comment]
Nice find and ❤️ for the quick turnaround!
